group.py

```diff
@@ -2,4 +2,60 @@
 
 # Your code to go here...
 
-my_group =
+my_group ={
+            "Jill": [26,"biologist",{"friend": "Zalika","partner": "John"}],
```

Comment:
maybe dictionary in dictionary is better because we can directly access the element without identify its index number in the list